### PR TITLE
Update the app with the current RS.js master and the new widget

### PR DESCRIPTION
**Pull request by gregkare**

### CODE REVIEW

File: remotestorage-module.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
-RemoteStorage.defineModule('myfavoritedrinks', function(privateClient) {
+// This constant will be reused by the other files
+// Enable change events for changes in the same browser window
+const remoteStorage = new RemoteStorage(
```
Comment (skddc) [inline remotestorage-module.js]: As this code is not being transpiled, we can't use ES6 syntax in here. It will break in a lot of browsers that way.

Comment (gregkare) [reply remotestorage-module.js]: Thanks!

File: style.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 html { background:#000; }
 body { position:relative; color:#fff; font:normal 1em FreeSans,Helvetica,sans-serif; }
 
-#wrapper { width:480px; padding:50px; margin:0; }
+#wrapper { font-size: 1.5em; width:480px; padding:50px; margin:0; }
```
Comment (skddc) [inline style.css]: The commit message doesn't say why this change was made, and what it results in exactly.

Comment (gregkare) [reply style.css]: I had set the font-size on body to 1em in the previous commit, this adds back the 1.5em font on a wrapper so the interface still has the same font size

Comment (skddc) [reply style.css]: Got it, thx.

File: remotestorage-module.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
-RemoteStorage.defineModule('myfavoritedrinks', function(privateClient) {
+// This variable will be reused in the other files
+// Enable change events for changes in the same browser window
```
Comment (skddc) [inline remotestorage-module.js]: This shouldn't be instantiated in the module file.

Comment (gregkare) [reply remotestorage-module.js]: Thanks, I forgot to write a note about this! Since this project is using JS files without a build system, I would instantiate the remoteStorage variable in a separate file that's loaded first. Is that right?

Comment (skddc) [reply remotestorage-module.js]: No, it should be done in the app code. And the module should be defined on `window.MyFavoriteDrinks` for example (that's what would happen with modules from npm as well).

Comment (gregkare) [reply remotestorage-module.js]: Ack, on it!

File: remotestorage-module.js [hunk 4]
```diff
@@ -1,41 +1,36 @@
...
-
-remoteStorage.addModule({name: 'myfavoritedrinks', builder: function(privateClient) {
-  privateClient.declareType('drink', {
-    type: 'object',
-    properties: {
-      name: { type: 'string' }
-    },
-    required: ['name']
-  });
+window.myfavoritedrinks = {
```
Comment (skddc) [inline remotestorage-module.js]: 1. Our convention here is UppercaseCamelCase.
2. You don't need to specify `window`. Any `var` outside of another scope will be attached to the global object, which is `window` in browsers.

**Comment by gregkare** [comment]
I have renamed the module

**Comment by gregkare** [comment]
I think I'm doing something wrong. In this PR when clicking the sync button it never stops turning, but I do get changes from the remote. @skddc @galfert Any idea?

**Comment by skddc** [comment]
Did you update both the widget and core library to at least RC2?

**Comment by gregkare** [comment]
Damn, I thought I did the widget was still on RC2. Updating the widget did the trick, thanks!

**Comment by skddc** [comment]
Has this actually been deployed? I didn't see any notifications...

**Comment by gregkare** [comment]
@skddc No, I hadn't deployed it yet, doing it now

**Comment by gregkare** [comment]
Update: I have deployed it
